grafast/website/src/pages/sponsor.mdx

```diff
@@ -35,16 +42,114 @@ sponsorship. THANK YOU!
     business
   />
 </Sponsors>
+<Sponsors level="leaders">
+  <Sponsor name="Robert Claypool" />
+  <Sponsor name="nigelrmtaylor" />
+  <Sponsor
+    name="Principia Mentis"
+    avatar="/images/sponsors/principiamentis.png"
+    business
+  />
+  <Sponsor
+    name="Trigger.dev"
+    avatar="/images/sponsors/triggerdev.png"
+    business
+  />
+  <Sponsor name="Axinom" avatar="/images/sponsors/axinom.png" business />
+  <Sponsor name="Taiste" avatar="/images/sponsors/taiste-sq.svg" business />
+  <Sponsor
+    name="BairesDev"
+    avatar="/images/sponsors/BairesDev.png"
+    href="https://www.bairesdev.com/sponsoring-open-source-projects/"
+    business
+  />
+  <Sponsor
+    name="Two Bit Solutions"
+    avatar="/images/sponsors/two-bits.svg"
+    business
+  />
+  <Sponsor name="Apollo" avatar="/images/sponsors/apollo.png" business />
+  <Sponsor name="Dimply" avatar="/images/sponsors/dimply.svg" business />
+  <Sponsor name="Ndustrial" avatar="/images/sponsors/ndustrial.png" business />
+  <Sponsor name="Cintra" avatar="/images/sponsors/cintra.svg" business />
+  <Sponsor name="Beacon" avatar="/images/sponsors/beaconbio.jpg" business />
+  <Sponsor
+    name="deliver.media"
+    avatar="/images/sponsors/deliverdotmedia.jpg"
+    business
+  />
+  <Sponsor name="Ravio" avatar="/images/sponsors/Ravio.svg" business />
+  <Sponsor name="prodready" />
+  <Sponsor name="Locomote" avatar="/images/sponsors/locomote.svg" />
+</Sponsors>
 <Sponsors level="backers">
+  <Sponsor name="plus 67 more backers..." plain />
   <Sponsor name="plus 84 more backers..." plain />
 </Sponsors>
 
-<!-- SPONSORS_END -->
+{/* SPONSORS_END */}
```

Comment:
Our script might need updating to handle this.

Reply:
Our script doesn't handle these websites yet anyway, but yes will need updating to handle the new way comments are made.